### DISCUSSION TITLE
Is it possible to add custom nodes in Animation Graph?

### DISCUSSION THREAD

**Discussion by firebeetleLobster** [o3de/o3de#10708] [Help Wanted]
I want to drive a morph target with the rotation of a bone in the Animation Graph,similar to the function of a 'driver' in blender.However,it seems that such a function can not be realized with existing nodes.The rotation data given by Get Transform Node seems to be quaternions and can not be decomposed into euler angles.
So ,is it possible to add a custom node to realize the function ,or have I missed something about Animation Graph?

**Comment by AMZN-alexpete** [comment]
@amzn-jillich @amzn-rhhong @burelc-amzn who likely know the answer to this
